### PR TITLE
UX Improvements

### PR DESCRIPTION
**Pull request by Naatan**

### CODE REVIEW

File: installers/install.sh [hunk 1]
```diff
@@ -168,6 +168,11 @@ if $FORCEOVERWRITE && ( ! $NOPROMPT ); then
   exit 1
 fi
 
+echo "\
+\033[2m╔═══════════════════════╗
```
Comment (mdrohmann) [inline installers/install.sh:172]: `2m` or `32m` like in the box below?

File: locale/en-us.yaml [hunk 2]
```diff
@@ -463,7 +461,7 @@ error_state_run_undefined_name:
 error_state_run_unknown_name:
   other: The script '{{.Name}}' is not defined in activestate.yaml.
 info_state_run_activating_state:
-  other: "[NOTICE]This script requires an activated state. Activating it now. To bypass this set `standalone: true` on this script in your activestate.yaml.[/RESET]"
+  other: "Activating runtime environment. To bypass this enter an activated state via `[ACTIONABLE]state activate[/RESET]` or if you do not require a runtime environment add `[ACTIONABLE]standalone: true[/RESET]` to script in your [NOTICE]activestate.yaml[/RESET]."
```
Comment (mdrohmann) [inline locale/en-us.yaml:464]:
```suggestion
  other: "Activating runtime environment. To bypass this enter an activated state via `[ACTIONABLE]state activate[/RESET]` or if you do not require a runtime environment add `[ACTIONABLE]standalone: true[/RESET]` to the script in your [NOTICE]activestate.yaml[/RESET]."
```

**Review by mdrohmann — CHANGES_REQUESTED** [review]
Just a few questions.  And I noticed that `install.ps1` is now UTF-16 encoded.  It is therefore also interpreted as a binary file by git.  Is it okay to change this?

Eg., `recode utf16..utf8 installers/install.ps1` worked for me.   I could not compare the changes otherwise...

File: cmd/state/errors.go [hunk 3]
```diff
@@ -32,22 +31,14 @@ func (o *OutputError) MarshalOutput(f output.Format) interface{} {
...
-	if len(errs) > 1 {
-		outLines = append(outLines, output.Heading(locale.Tl("err_what_happened", "Here's What Happened")).String())
+	outLines = append(outLines, output.Heading(locale.Tl("err_what_happened", "[ERROR]Something Went Wrong[/RESET]")).String())
 
-		for _, errv := range errs[1:] {
-			outLines = append(outLines, fmt.Sprintf(" [ERROR]x[/RESET] %s", trimError(locale.ErrorMessage(errv))))
-		}
+	errs := locale.UnwrapError(o.error)
+	for _, errv := range errs {
+		outLines = append(outLines, fmt.Sprintf(" [NOTICE][ERROR]x[/RESET] %s", trimError(locale.ErrorMessage(errv))))
```
[the remark this answers is not in the document]
Comment (Naatan) [reply cmd/state/errors.go:41]: Yes and yes

File: installers/install.sh (hunk 1 above)
Comment (Naatan) [reply installers/install.sh:172]: `2m`. The box below is a success state so its outline is green.

**Comment by mdrohmann** [comment]
I think you missed this one:
> [...] I noticed that `install.ps1` is now UTF-16 encoded. It is therefore also interpreted as a binary file by git. Is it okay to change this back to utf-8?
> 
> Eg., `recode utf16..utf8 installers/install.ps1` worked for me. I could not compare the changes otherwise...

**Comment by Naatan** [comment]
@mdrohmann hmm I'll file a story to follow up on that, but for now lets go with this. UTF-8 causes issues with the title box formatting in powershell. I'm not sure what the exact issue is though.

**Review by mdrohmann — APPROVED** [review]

**Pull request merged**